bundles/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoPlugHandler.java

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.neohub.internal;
+
+import static org.openhab.binding.neohub.internal.NeoHubBindingConstants.*;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+
+/**
+ * The {@link NeoPlugHandler} is the OpenHab Handler for NeoPlug devices Note:
+ * inherits almost all the functionality of a {@link NeoBaseHandler}
+ * 
+ * @author Andrew Fiddian-Green - Initial contribution
+ * 
+ */
+public class NeoPlugHandler extends NeoBaseHandler {
+
+    public NeoPlugHandler(Thing thing) {
+        super(thing);
+    }
+
+    // =========== methods of NeoBaseHandler that are overridden ================
+
+    /*
```

Comment:
remove this comment - the method has JavaDoc in the super class

Reply:
Ok, done.